### PR TITLE
Also respond to 'simpsons me'

### PR DESCRIPTION
**Pull request by tabacco**
The "[action] me" phrasing is more consistent with other hubot scripts (and is also less stuff to type if you're lazy, like me)

### CODE REVIEW

**Comment by okize** [comment]
@tabacco I considered doing this but I was worried that there could be a conflict with existing simpsons-themed hubot plugins. 

I would personally prefer this but let me do a little due diligence before merging this PR.

**Comment by tabacco** [comment]
Hmm... Maybe a shorter version like 'frink me'?

**Comment by okize** [comment]
So, these are the only two hubot plugins I've found (and they appear to be the same plugin):
- https://github.com/hubot-scripts/hubot-thesimpsons
- https://github.com/omardelarosa/hubot-simpsons

The command there is `hubot simpsons <quote | image>` so I don't think this PR will conflict. 

Nonetheless I'll bump the major version for this and if there ends up being a conflict somewhere the 1.x version could still be used.

File: src/frinkiac.coffee [hunk 1]
```diff
@@ -80,7 +80,7 @@ combineCaptions = (captions) ->
     captions[0].Content
 
 module.exports = (robot) ->
-  robot.respond /(simpsons search|frinkiac) (.*)/i, (msg) ->
+  robot.respond /(simpsons search|frinkiac|me) (.*)/i, (msg) ->
```
Comment (okize) [inline src/frinkiac.coffee]: Hmmm, I believe this would match `hubot me`. Perhaps `/(simpsons (search|me)|frinkiac) (.*)/` would give you what you want?

Comment (tabacco) [inline src/frinkiac.coffee]: Ah yes, I regex'd wrong, sorry about that.

Comment (okize) [inline src/frinkiac.coffee]: Looks good & thanks for adding a test!

**Comment by okize** [comment]
Published this PR in v2.0.0

**Pull request merged**